src/SampleApplication/Program.cs

```diff
@@ -11,54 +11,100 @@ namespace SampleApplication
 {
     public static class Program
     {
+        const string SampleApplicationAssemblyPath = "SampleApplication\\bin\\Debug\\net6.0\\SampleApplication.dll";
+        const string SampleWithSuffixV4AssemblyPath = "Samples\\FromSufix\\v4\\bin\\Debug\\net6.0\\SampleScripts.dll";
+        const string SampleWithSuffixV5AssemblyPath = "Samples\\FromSufix\\v5\\bin\\Debug\\net6.0\\SampleScripts.dll";
+        const string SampleWithFolderV4AssemblyPath = "Samples\\FromFolder\\v4\\bin\\Debug\\net6.0\\SampleScripts.dll";
+        const string SampleWithFolderV5AssemblyPath = "Samples\\FromFolder\\v5\\bin\\Debug\\net6.0\\SampleScripts.dll";
+
         public static void Main(string[] args)
         {
-            string instanceName = @".\SqlExpress";
-            // Uncomment the following line to run against sql local db instance.
+            string instanceName = @".";
```

Comment:
I had to use this to get it to work with LocalDb: `@"(LocalDB)\MSSQLLocalDB"`. Probably not important for this change though.